Corzbank.Web/ClientApp/src/app/header/wallet/wallet.component.html

```diff
@@ -1,46 +1,44 @@
 <div class="wallet">
 	<div class="cards-deposits">
-		<div
-			class="option show-cards-title"
-			(click)="cardsIsDisplayed = !cardsIsDisplayed"
-		>
+		<div class="option show-cards-title" (click)="cardsIsDisplayed = !cardsIsDisplayed">
 			<div class="selected-option" *ngIf="cardsIsDisplayed"></div>
 			<div class="option-title">
+				<hr />
 				<h2>Cards</h2>
 				<hr />
 			</div>
 		</div>
 		<div class="cards" *ngIf="cardsIsDisplayed">
-			<div
-				class="card"
-				[ngClass]="{
+			<h1 *ngIf="cards.length === 0" class="none-items">
+				You do not have any cards
+			</h1>
+			<div class="card" [ngClass]="{
 					credit: card.cardType == 1,
 					debit: card.cardType == 2,
 					universal: card.cardType == 3
```

Comment:
1. card.cardType == 1 is a bad approach. Card type might change in future, and value 1 will not reffered to the same enum property. Rewrite to card.cardType == CardType.Value

Reply:
resolved